doc/examples/example4.md

```diff
@@ -0,0 +1,79 @@
+Certificate Loading  {#example4}
```

Comment:
Change this title to something more generic. Like "Working with X509 certificates"